File: GrupoSistemas/GrupoSistemas/MvcApplication1/Clases/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace MvcApplication1.Clases
{
    public class Cliente
    {
        public string CodCliente { get; set; }
        public string Nombre { get; set; }
        public string ApePat { get; set; }
        public string ApeMat { get; set; }
        public string Direccion { get; set; }
        public int Telefono { get; set; }
        public string Correo { get; set; }
        public string EstadoCivil { get; set; }
        public DateTime FechaNacimiento { get; set; }
    }
}

File: GrupoSistemas/GrupoSistemas/GrupoSistemas/Clases/Cliente.vb
Public Class Cliente

    Private _CodCliente As String
    Private _Nombre As String
    Private _ApePat As String
    Private _ApeMat As String
    Private _Direccion As String
    Private _Telefono As Integer
    Private _Correo As String
    Private _EstadoCivil As String
    Private _FechaNacimiento As Date

    Public Property CodCliente() As String
        Get
            Return _CodCliente
        End Get
        Set(ByVal value As String)
            _CodCliente = value
        End Set
    End Property

    Public Property Nombre() As String
        Get
            Return _Nombre
        End Get
        Set(ByVal value As String)
            _Nombre = value
        End Set
    End Property

    Public Property ApePat() As String
        Get
            Return _ApePat
        End Get
        Set(ByVal value As String)
            _ApePat = value
        End Set
    End Property





    Sub New(ByVal v_idArea As Byte)
        Me._idArea = v_idArea
    End Sub

    Sub New()

    End Sub



End Class
